core/src/Config.purs

```diff
@@ -101,54 +101,54 @@ type PublishConfig =
   }
 
 publishConfigCodec :: JsonCodec PublishConfig
-publishConfigCodec = CAR.object "PublishConfig"
-  { version: Version.codec
-  , license: License.codec
-  , location: CAR.optional Location.codec
-  , include: CAR.optional (CA.array CA.string)
-  , exclude: CAR.optional (CA.array CA.string)
-  }
+publishConfigCodec = CA.object "PublishConfig"
+  $ CA.recordProp (Proxy :: _ "version") Version.codec
+  $ CA.recordProp (Proxy :: _ "license") License.codec
+  $ CA.recordPropOptional (Proxy :: _ "location") Location.codec
+  $ CA.recordPropOptional (Proxy :: _ "include") (CA.array CA.string)
+  $ CA.recordPropOptional (Proxy :: _ "exclude") (CA.array CA.string)
+  $ CA.record
 
 type RunConfig =
   { main :: Maybe String
-  , execArgs :: Maybe (Array String)
+  , exec_args :: Maybe (Array String)
```

Comment:
These (in RunConfig and TestConfig) are the only snake cases we need, it should go in camel case in the rest of the codebase